go.mod

```diff
@@ -7,14 +7,15 @@ require (
 	github.com/BurntSushi/toml v1.4.1-0.20240526193622-a339e1f7089c
 	github.com/alecthomas/jsonschema v0.0.0-20220216202328-9eeeec9d044b
 	github.com/benbjohnson/clock v1.3.5
-	github.com/celestiaorg/celestia-app/v2 v2.2.0
+	github.com/celestiaorg/celestia-app/v3 v3.0.0-alpha
```

Comment:
we have v3.0.0-rc0 now

tried to open a PR, but didn't want to open on @vgonkivs 's fork and github doesn't keep track of that fork for some reason

Reply:
I will bump it here. Thanks @evan-forbes